tests/test_route.py

```diff
@@ -5,81 +5,115 @@
 import pytest
 
 from swsscommon import swsscommon
+from dvslib.dvs_common import wait_for_result
+from dvslib.dvs_database import DVSDatabase
 
-
-class TestRoute(object):
+class TestRouteBase(object):
     def setup_db(self, dvs):
-        self.pdb = swsscommon.DBConnector(0, dvs.redis_sock, 0)
-        self.adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
-        self.cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+        self.pdb = dvs.get_app_db()
+        self.adb = dvs.get_asic_db()
+        self.cdb = dvs.get_config_db()
 
     def set_admin_status(self, interface, status):
-        tbl = swsscommon.Table(self.cdb, "PORT")
-        fvs = swsscommon.FieldValuePairs([("admin_status", status)])
-        tbl.set(interface, fvs)
+        self.cdb.update_entry("PORT", interface, {"admin_status": status})
         time.sleep(1)
```

Comment:
Do you know if these `sleep` calls are still needed with all of the new polling functions you added?